### ISSUE TITLE
TodoParser: ArrayIndexOutOfBoundsException when parsing IllegalTodos.

### ISSUE THREAD

**Issue by criske** [self-xdsd/todo-finder-cli#8]
Throws instead of returning empty list.
```java
java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1

	at com.selfxdsd.todocli.TodoParser.setRemainingTodoFields(TodoParser.java:167)
	at com.selfxdsd.todocli.TodoParser.parse(TodoParser.java:116)
	at com.selfxdsd.todocli.TodoParserTest.illegalTodosClassExpectNoTodos(TodoParserTest.java:195)
```
Reproduce by adding this
```java
/**
 * TODO foo
 */
```
 in `IllegalTodos.java` test file and run test  `TodoParserTest.illegalTodosClassExpectNoTodos`.
(for lowercase "todo" works fine).

**Comment by charlesmike** [comment]
@criske thank you for reporting this. I'll assign someone to take care of it soon.

**Comment by charlesmike** [comment]
@criske this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2020-11-22T12:07:20.871113``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it partially and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

**Comment by charlesmike** [comment]
@criske thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).
